eq-author-api/schema/resolvers/base.js

```diff
@@ -225,63 +218,49 @@ const Resolvers = {
 
   Mutation: {
     createQuestionnaire: async (root, args, ctx) => {
-      const questionnaire = createQuestionnaire({
+      const questionnaire = createNewQuestionnaire({
         ...args.input,
         createdBy: ctx.auth.name,
       });
-      save(questionnaire);
-      const questionnaireList = getQuestionnaireList();
-      questionnaireList.push({
-        ...omit(questionnaire, "sections", "metadata"),
-        createdAt: questionnaire.createdAt.toString().split("T")[0],
-      });
-      saveQuestionnaireList(questionnaireList);
+
+      createQuestionnaire(questionnaire);
```

Comment:
I presume we should await here?

Reply:
even though there is an await inside `createQuestionnaire`?

Reply:
Yeah, `async` functions return a `Promise` so to wait for them to be done you have to `await` them otherwise you'll start the outer one but not await on its result even if internally it was waiting on something else to finish. I hope that makes sense.